### DISCUSSION TITLE
Tower Defense - Basic Information

### DISCUSSION THREAD

**Discussion by TimWaeckerle** [Freaky2164/TowerDefense#3] [General]
### Team Tower Defense!

Lukas Weber --- Project Owner/Architect
Paul Faller --- Scrum Master/Design Reviewer
Tim Wäckerle --- Designer
Johannes Methfessel --- Technical Lead
Nico Argast --- Designer

Basic Role for everyone: Developer

### **Our Project**

We are going to create a simple Tower Defense game that runs on Android Phones. 
The map should be split in 2 Parts:
1. The way through the map where enemies walk to your base to decrease your HP
2. A place where you can place your turrets to defend your base!

We may add some features during the Sprints and are open for your feature requests!

### **Platforms:**

1. GitHub as version control tool
2. GitHub Discussions for our blog
4. Youtrack to plan our Sprints
5. Unity as game engine (C#)

### **Scrum Setup**

Software: Youtrack
Sprint length: 2 weeks
Sprint Meeting: every second Friday
Dailys: Mo. Tu. Fri. 

### Sprint Board #1

![image](https://user-images.githubusercontent.com/97874901/196670087-2999b6b1-4c34-4af5-8571-3ef886275f9a.png)

### Backlog

![image](https://user-images.githubusercontent.com/97874901/196676641-14b3256e-06c1-4d44-a5c6-68e7c3369c79.png)

**Comment by Mohmad-Naser-alnakeshbandi** [comment]
Hey everyone 🖐

The project sound amazing, 

I am really interesting in "Game sharing" as a feature in Tower Defense,  so for example: 
Someone might send a link of his game, and I could see how to win the 3rd level. 

Are you planning to develop such a feature?

Looking forward to reading more from you ✨

**Reply by TimWaeckerle** [reply]
Hi,

Thanks for your contribution! Your idea sounds very interesting.

Since we are planning an offline app, i don't think we will have enough time to add such a feature.
But we will discuss again and think about it.

I hope you continue to support our development and enjoy our game when it's finished

**Comment by Yolohans** [comment]
Hey,
great project, I love tower defences! Are you planning to set up a map editing framework that you will share for users to create their own maps?
Do you plan to have a theme that all your map/tower designs follow, eg. medieval/fantasy, scify or steampunk?

**Reply by TimWaeckerle** [reply]
Hi,
we have planned this as an optional feature that will only be implemented if there is time left.
We still have to agree on the exact design internally.

Thanks for your contribution!

TD Team

**Comment by alex020302** [comment]
Hello Team Tower Defense,

I am a member from DhbWorkout. As a kid I loved tower defense games, 
like Bloons TD or Plants vs Zombies. That's why I find your project very interesting. 

I'm interested in the reasons why you decided to make an Android app. 
Do any of you already have experience with app development and 
are there any advantages or easier development environment compared to a Java app?
Do you plan to design a part of the game yourself, like enemies or towers and 
do you have some kind of theme for your game (monsters, aliens or something like that)?
Do you plan to make your project available for download at the end of the academic year?

We would be happy about your answer and wish you good luck in the realization of your project.

Kind regards,
Team DHBWorkout

**Reply by TimWaeckerle** [reply]
Hey!

1. One of our members already has experience with android apps. Also its way more effort to install a self made app on an IOS Phone which made us choose android as platform. We are using C# because we use Unity.
2. We still have to agree on the exact design internally.
3. We have discussed about adding the app to the Playstore and have decided to include it depending on the result.


Thanks for your contribution!

TD Team

**Comment by jan-nie** [comment]
Hello team towerdefense,

your vision sounds really interesting and we are looking forward to see first results and play your game.
Do you already know if and which frameworks you want to use for the implementation?
Have you already created a mockup that gives us first insights into the application?
Is there a reason why you don't support iOS devices?
Will you make the game dynamic so that different screen sizes are supported?

We wish you much success in the implementation and will follow your process with interest.

Best wishes
Team Framgång

**Reply by argastle** [reply]
Hello,

Question 1-3 is already answerd in the text above.
The 4th. is defined by Unity. We will not add extra workload to get it running on multiple divices

~ TowerDefence Team
